release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -119,6 +119,11 @@ For more information, see xref:../installing/installing_bare_metal_ipi/ipi-insta
 
 For more information, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#configuring-the-bios_ipi-install-installation-workflow[Configuring the BIOS] and xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#configuring-the-raid_ipi-install-installation-workflow[Configuring the RAID].
 
+[id="ocp-4-11-heterogeneous-tech-preview"]
+==== {product-title} on Heterogeneous architectures (Technology Preview)
+{product-title} 4.11 introduces Heterogeneous architecture cluster support using Azure installer-provisioned infrastructures in Technology preview.
+This feature offers the ability to add  `arm64` or `x86_64`  worker nodes to an existing Azure cluster with a `x86_64` control plane as a day two operation. Control planes on `arm64` architectures are not currently supported.
```

Comment:
i think we should  make it clear here that the cluster should be provisioned as a "single" arch cluster (homogeneous) using the normal IPI process but with an installer made available here: https://mirror.openshift.com/pub/openshift-v4/multi/clients/ocp-dev-preview/ 

once that is done, worker nodes of any architecture can be added.

Reply:
@Prashanth684 Hi Prashanth! Just a question, so the Azure IPI install for heterogeneous is the same, except that when you deploy the cluster, you must override the release image and use the mixed-arch installer that you added above?

Reply:
if you use the installer binary from the mirror link i posted above, you should not need to override the image. it will  install with the correct heterogeneous payload.